cobc/parser.y

```diff
@@ -3201,6 +3233,9 @@ set_record_size (cb_tree min, cb_tree max)
 %token YYYYMMDD
 %token ZERO
 
+%token LEVEL_NUMBER_IN_AREA_A	"level-number (in Area A when relevant)"
```

Comment:
I did that but bison complains about duplicate identifier strings. Also I noted recent versions of  bison have some internationalization support (but alas, versions not even available in the CI here).

Reply:
It's only warnings though…

Reply:
… but then it uses `LEVEL_NUMBER_IN_AREA_A` in error messages.
All this could be improved by redefining the syntax error printing function to intercept those tokens, but that's a bit too much work for a short-term upstreaming.

Reply:
Agreed, so let's go with the "pure" name + " (Area A)".

Reply:
Ok, better keep it short indeed.